plugins/inputs/iptables/README.md

```diff
@@ -0,0 +1,48 @@
+# Iptables Plugin
+
+The iptables plugin gathers packets and bytes counters for rules within a set of table and chain from the Linux's iptables firewall.
+
+The iptables command requires root access. You can run telegraf as root (not recommended), or you
+can configure telegraf to use `sudo` for running iptables.
```

Comment:
Excellent, I will mention this in the doc/config. While sudo allows to grant less rights, CAP_NET_ADMIN makes it easier to configure through the systemd service configuration file. Thanks !